### ISSUE TITLE
"login" and "trySilentLogin" only can trigger once after "login success"

### ISSUE THREAD

**Issue by bau720123** [EddyVerbruggen/cordova-plugin-googleplus#45]
hi @EddyVerbruggen 

when I use
```
window.plugins.googleplus.login
```
it will fully success in Android and iOS
and got many information about JSON.stringify(obj)

well...after this
when I use these method again
```
window.plugins.googleplus.login
OR
window.plugins.googleplus.trySilentLogin
```

[in iOS]
it will still got many information about JSON.stringify(obj) again

but...
[in Android 4.4.2]
it will no any response anymore

**Comment by romanostolosh-devpronet** [comment]
I can confirm that.. Same behavior for me

**Comment by bau720123** [comment]
hi @EddyVerbruggen 
anything update or ?

**Comment by EddyVerbruggen** [comment]
Hi guys, found a solution. I previously used `connect` to open a connection to Google Play Services. I changed this to `reconnect` which always opens a new connection, forcing the `onConnected` delegate method to be invoked (which wasn't the case when `connect` was used and the app was already connected).

**Comment by bau720123** [comment]
hi @EddyVerbruggen 
thanks for your reply
so....in version 1.1.1 that be can directly use "reconnect" to instead "connect" ?

**Comment by bau720123** [comment]
hi all
problem solved in version 1.1.1
thanks @EddyVerbruggen
